v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/transforms/WriteToElasticsearch.java

```diff
@@ -125,6 +125,12 @@ public PDone expand(PCollection<String> jsonStrings) {
       config = config.withApiKey(options().getApiKey());
     }
 
+    
+    if (options().getTrustSelfSignedCerts()!= null) {
```

Comment:
Please run `mvn spotless:apply`, this is breaking some of the formatting rules

Reply:
sure thing